packages/@sanity/desk-tool/src/panes/document/DocumentPane.tsx

```diff
@@ -1,4 +1,4 @@
-import {useDocumentType} from '@sanity/base/hooks'
+import {unstable_useTemplatePermissions, useDocumentType} from '@sanity/base/hooks'
```

Comment:
i still think we should aliases these imports to make them register with the hooks linter.

e.g.
 
```js
function Comp() {
  if (cond) return null

  // 🔴 this will not be caught by the hooks linter
  const x = unstable_useTemplatePermissions()
}
```